apps/client/src/components/Conclusion.js

```diff
@@ -113,6 +114,8 @@ const Conclusion = ({ checker, matomoTrackEvent }) => {
             </PrintButton>
           </ComponentWrapper>
         )}
+
+        <ConclusionModal />
```

Comment:
It's not the conclusion that's in the modal.

Reply:
Check! Thanks